osvvm-utility.core

```diff
@@ -1,5 +1,21 @@
 CAPI=2:
 
+#  This file is part of OSVVM.
+#
+#  (c) Crown Copyright 2020
```

Comment:
Can you write this in the form:
Copyright (c) 2020 by Crown

Reply:
Sure. Done.